### PR TITLE
My review

### PR DESCRIPTION
**Pull request by mike2611**
This project was made by Fabrizio and me. The goal of this project is to test my skills as a code reviewer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# project-review
\ No newline at end of file
+# project-FabrizioMiguel
+
+link to the page ->  https://mike2611.github.io/project-FabrizioMiguel/
```
Comment (mike2611) [inline README.md]: Please make your README file more descriptive you can add the authors for the project a screenshot of it. And please also use GitHub markdown to make your file more legible. 
A great README file makes your projects look a lot more professional. :smile:

Comment (mike2611) [reply README.md]: It would also be good to add links to your partner's LinkedIn and GitHub and yours

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#image{
+    display: block;
+    margin:0 auto;
+    max-width: 60%;
+    
+}
```
Comment (mike2611) [inline css/style.css:6]: Please make sure to follow the linter rules. for example in this case 
#image {
  display: block;
  margin:0 auto;
  max-width: 60%;
}

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <title id="title">Elvis Presley</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body id="main">
+    <div id="img-div">
+      <img id="image" src="https://pymstatic.com/84696/conversions/frases-elvis-presley-wide.jpg">
+      <p id="img-caption" > “Values are like fingerprints. Nobody's are the same, but you leave 'em all over everything you do” </p>
+    </div>
+    <div id="tribute-info">
```
Comment (mike2611) [inline index.html:14]: This tag should be a <section></section> please make sure to use semantic tags . Semantic elements help search engines to read the page and find the required information faster also it's useful for people with vision problems.

**Review by mike2611 — COMMENTED** [review]
Changes required :smiley: please check the comments that I left in order to make your project look more professional. 

Don't hesitate to ask any doubts you may have.
Thanks have a nice day.

File: css/style.css (hunk 2 above)
Comment (henatan99) [reply css/style.css:6]: - [ ] Nice review.It is polite and suggestive. I think you can make more description why he should keep linter rules. For example, you can say ' linter rules can make your code more clean and readable'.

File: index.html (hunk 3 above)
Comment (henatan99) [reply index.html:14]: - [ ] The first sentence is a bit vague. Isn't it better if it is written as 'This tag should be an `a` tag'?
- [ ] Please use bullets with tick box ` - [ ] ` which helps mark resolved comments as I did in this comment.
- [ ] OPTIONAL: Using emojis and/or gifs makes your comment more human-friendly. :robot: :alien: Kindly, try some of them.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @mike2611 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mike2611 — COMMENTED** [review]
I edited my comment pleases checked for any new information :recycle: 

Happy coding. Have a nice day :smile:

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @mike2611 😃, 

Well done delivering this great review and working on the requested changes ☺️ Also, big thanks for using emojis 😉
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**